announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-metadata.vm

```diff
@@ -1,5 +1,14 @@
 <!-- $Header: /cvs/sakai2/legacy/tools/src/webapp/vm/announcement/chef_announcements-metadata.vm,v 1.7 2005/05/15 17:02:52 gsilver.umich.edu Exp $ -->
 
+<script type="text/JavaScript">
+<!--
```

Comment:
OK, but this is existing code that had accidentally been removed. I just put it back.